src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -235,13 +236,24 @@ const VisitorSpaceSearchPage: NextPage = () => {
 	 * Methods
 	 */
 
+	const prepareSearchValue = (value = '') => {
+		if (value.trim() && !query.search?.includes(value)) {
+			return {
+				[SEARCH_QUERY_KEY]: (query.search ?? []).concat(value),
+			};
+		}
+
+		return undefined;
```

Comment:
personal preference: 
When i set something explicitly i prefer to use null.
when something is missing without setting it explicitly it is undefined.

Reply:
best to illustrate why I prefer `undefined` over `null` across the board.
<img width="255" alt="image" src="https://user-images.githubusercontent.com/90180417/191730612-1243bfee-ee37-4239-9341-4a7b1b208f73.png">

Reply:
```
let nullable: { prop?: string | null }
```